gourmet/keymanager.py

```diff
@@ -56,9 +56,9 @@ def make_regexp_for_strings (self, ignored):
 
     def regexp_for_all_words (self, txt):
         """Return a regexp to match any of the words in string."""
-        regexp="(^|\W)("
+        regexp=r"(^|\W)("
         count=0
-        for w in re.split("\W+",txt):
+        for w in re.split(r"\W+",txt):
```

Comment:
Here, `word_splitter` could be used, no?

Reply:
Sure, thanks!